packages/react-scripts/utils/createJestConfig.js

```diff
@@ -20,8 +20,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     moduleNameMapper: {
-      '^.+\\.(?!(js|jsx|css|json)$)[^\\.]+$': resolve('config/jest/FileStub.js'),
```

Comment:
does this not already exclude `.css` files? It matches any other extension (or thing that looks like an extension!) -  is this an alternative to adding `.less` etc to the exclusion list?